media/themes/Mizu/mizu.theme

```diff
@@ -0,0 +1,12 @@
+
+[Theme]
+DisplayName=ReactOS Mizu
+
+[VisualStyles]
+Path=%WinDir%\Resources\themes\mizu\mizu.msstyles
+ColorStyle=NormalColor
+Size=NormalSize
+
+[MasterThemeSelector]
+MTSM=DABJDKT
+;; Use RJSPBS for Win8+
```

Comment:
Some cryptic stuff here

Reply:
"Not My Fault" (TM) :joy:
https://learn.microsoft.com/en-us/windows/win32/controls/themesfileformat-overview#masterthemeselector-section
but:
https://forums.mydigitallife.net/threads/multi-oem-retail-project-mrp-mk3.71555/page-306#post-1609621